frontend/public/components/utils/dropdown.jsx

```diff
@@ -150,6 +151,13 @@ class DropDownRow extends React.PureComponent {
             e.stopPropagation();
             onBookmark(itemKey, !isBookmarked);
           }}
+          aria-label={
+            _.isString(content)
+              ? isBookmarked
+                ? t('dropdown~Remove bookmark {{content}}', { content })
+                : t('dropdown~Add bookmark {{content}}', { content })
+              : null
+          }
```

Comment:
Like the `isString` check. I directly thought "but what happen if the content is a react node."

I think you could still use the aria-label `"Remove bookmark "` if the content is not a string. This would help more than no aria-label in this case. So maybe:

```suggestion
          aria-label={
            isBookmarked
              ? t('dropdown~Remove bookmark {{content}}', { _.isString(content) ? content : '' })
              : t('dropdown~Add bookmark {{content}}', { _.isString(content) ? content : '' })
          }
```

Reply:
not sure how I added this commit... sorry